c/python/namedtuple.cc

```diff
@@ -0,0 +1,79 @@
+//------------------------------------------------------------------------------
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// © H2O.ai 2018
+//------------------------------------------------------------------------------
```

Comment:
how about MIT?

Reply:
Guess it came from your `tuple.cc` file that I used as a reference. Will change it shortly. Btw, what about if we just replace it automatically in all the relevant files, so that we have a consistent license in all the places?